sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -262,6 +264,22 @@ textAnalyticsClient.recognizeEntities(document).forEach(entity ->
 For samples on using the production recommended option `RecognizeEntitiesBatch` see [here][recognize_entities_sample].
 Please refer to the service documentation for a conceptual discussion of [named entity recognition][named_entity_recognition].
 
+### Recognize Personally Identifiable Information entities
+Run a predictive model to identify a collection of Personally Identifiable Information(PII) entities in the passed-in 
+document. It recognizes and categorizes PII entities in its input text, such as
+Social Security Numbers, bank account information, credit card numbers, and more. This endpoint is only available for 
+v3.1-preview.1 and up.
+
+<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L158-L161 -->
+```java
+String document = "My SSN is 555-55-5555";
+textAnalyticsClient.recognizePiiEntities(document).forEach(entity -> System.out.printf(
+    "Recognized Personally Identifiable Information entity: %s, entity category: %s, entity subcategory: %s, offset: %s, length: %s, confidence score: %f.%n",
+    entity.getText(), entity.getCategory(), entity.getSubcategory(), entity.getOffset(), entity.getLength(), entity.getConfidenceScore()));
+```
+For samples on using the production recommended option `RecognizePiiEntitiesBatch` see [here][recognize_pii_entities_sample].
+Please refer to the service documentation for [supported PII entity types][pii_entity_recognition].
```

Comment:
Do we do this section for all the readme snippets if not consider moving this to `Next Steps`?

Reply:
Not getting what you mean here. It should already for all the README snippets. It followes the pattern.

Reply:
I suggested if this could be moved to Next Steps section.